src/charm.py

```diff
@@ -262,9 +288,13 @@ def _get_sans_dns(self) -> FrozenSet[str]:
             return frozenset(config_sans_dns.split(","))
         mode = self._get_config_mode()
         if mode == Mode.UNIT:
-            return frozenset(["{self.app.name}-{self._get_unit_number()}.{self.model.name}"])
+            return frozenset(
+                [
+                    f"{self.app.name}-{self._get_unit_number()}-{certificate_number}.{self.model.name}"
```

Comment:
could this be a bit more descriptive? 
Instead of
```
tls-certificates-requirer-0-0.demo
```
it could be
```
tls-certificates-requirer-unit-0-certificate-0.demo
```

Reply:
I went ahead with:
- `unit`: `cert-<certificate number>.unit-<unit number>.<app name>.<model name>`
- `app`: `cert-<certificate number>.<app name>.<model name>`